wd_core.au3

```diff
@@ -1490,8 +1490,8 @@ EndFunc   ;==>__WD_Delete
 ; Syntax ........: __WD_Error($sWhere, $iErr[, $sMessage = Default[, $iExt = Default]])
 ; Parameters ....: $sWhere     - Name of calling routine
 ;                  $iErr       - The error number from the calling function
-;                  $sMessage   - Message that will be passed to the console/output [optional]
-;                  $iExt       - Extended information from the calling function [optional]. Default is 0
+;                  $sMessage   - [optional] Message that will be passed to the console/output
+;                  $iExt       - [optional] Extended information from the calling function
```

Comment:
Why `Default is 0` was removed ?

Reply:
`			Local $sExtended = (($iExt <> Default) ? (" / " & $iExt) : (""))`

Reply:
[__WD_Error EXT fixed](https://github.com/Danp2/au3WebDriver/pull/290/commits/37e04f327f216141e676f99a418c1322ddf322c6)

Reply:
So you no longer want to optionally display @extended?

Reply:
Why not?
Current version in this PR is:
```autoit
Local $sExtended = (($iExt) ? (" / " & $iExt) : (""))
```
So it will be showed when $iExt is > 0

Reply:
Ok. I must have looked at it incorrectly. There's still a change in behavior in that the prior version would display 0 for @extended where the proposed version doesn't. This is why I took out `Default is 0` previously.

Reply:
If you have any idea for a change here, please suggest it by changing the code.

Reply:
Please go back and review my changes [here](https://github.com/Danp2/au3WebDriver/pull/290/commits/61c1b5a7344f50e9be235d31058a0efbff8361d0). With this implementation, the @extended value was displayed only when submitted to __WD_Error, even if it was zero.

Reply:
I'm not sure with this. What you mean?

Reply:
We seem to be going in circles. I already tried to address this with you above --

> There's still a change in behavior in that the prior version would display 0 for @Extended where the proposed version doesn't. This is why I took out Default is 0 previously.

I tried to revise your changes by eliminating the default=0 portion so that you could still write something like --

`Local $sExtended = (($iExt <> Default) ? (" / " & $iExt) : (""))`

Your changes prevent the use of `Default` like above.

Reply:
Currently `wd_core.au3` from this PR have this part of code:

```autoit
Func __WD_Error($sWhere, $iErr, $sMessage = Default, $iExt = Default)
	Local Const $sFuncName = "__WD_Error"
	Local $sMsg

	If $sMessage = Default Then $sMessage = ''
	If $iExt = Default Then $iExt = 0

	Switch $_WD_DEBUG
		Case $_WD_DEBUG_None

		Case $_WD_DEBUG_Error
			If $iErr <> $_WD_ERROR_Success Then ContinueCase

		Case $_WD_DEBUG_Info, $_WD_DEBUG_Full
			Local $sExtended = ($iExt) ? (" / " & $iExt) : ("")
```

Reply:
Oh I see.
You want to keep showing $iExt on case when it is 0 only when i is Default then should not be put to the log.

Do I understand it correctly ?

Reply:
> You want to keep showing $iExt on case when it is 0 only when i is Default then should not be put to the log.

Exactly... because that is how it was working prior to your PR. 😉

Reply:
Rewording:
You want to keep showing $iExt on case when it is 0, and only when $iExt is Default then it should not be put to the log.

Reply:
Take a look:
[__WD_Error() $iExt Default behavior](https://github.com/Danp2/au3WebDriver/pull/290/commits/0d780b2beeb77aa7acbc4c916b7ef885b16bf3c6)